AutoRest/Generators/Java/Java/JavaCodeNamer.cs

```diff
@@ -348,7 +348,7 @@ public static String ImportedFrom(PrimaryType primaryType)
             else if (primaryType == PrimaryType.TimeSpan ||
                 primaryType.Name == "Period")
             {
-                return "java.time.Period";
+                return "org.joda.time.Period";
```

Comment:
Cool! Thanks for the detailed explanation. I also think `Period` is good at handling daylight saving time and less buggy. (from [this post](http://jnb.ociweb.com/jnb/jnbJul2008.html)) In worst cases, people who don't like `Period` can always call `toDuration`.